8Knot/queries/commits_query.py

```diff
@@ -1,16 +1,11 @@
 import logging
-from db_manager.augur_manager import AugurManager
 from app import celery_app
-import pandas as pd
-from cache_manager.cache_manager import CacheManager as cm
-import io
-import datetime as dt
-from sqlalchemy.exc import SQLAlchemyError
+import cache_manager.cache_facade as cf
 
 # DEBUGGING
 import os
 
-QUERY_NAME = "COMMITS"
+QUERY_NAME = "commits_query"
```

Comment:
For all query files: update to new format not using QUERY_NAME

Reply:
got it, switched everything over.